flashlight/app/imgclass/examples/ImageNetTransformer.cpp

```diff
@@ -282,18 +302,23 @@ int main(int argc, char** argv) {
       opt.zeroGrad();
 
       // Make a Variable from the input array.
-      Variable inputs;
+      Variable inputs = noGrad(example[kImagenetInputIdx]);
       float mixP =
           static_cast<float>(std::rand()) / static_cast<float>(RAND_MAX);
-      float lambda =
-          static_cast<float>(std::rand()) / static_cast<float>(RAND_MAX);
-      float example1Weight = FLAGS_train_p_mixup;
+      float example1Weight = 0.;
       if (FLAGS_train_p_mixup > 0 && mixP > 0.5) {
         // mixup
-        inputs = FLAGS_train_p_mixup * noGrad(example1[kImagenetInputIdx]) +
-            (1 - FLAGS_train_p_mixup) * noGrad(example[kImagenetInputIdx]);
-      } else {
+        float lambda = FLAGS_train_p_mixup - 0.2 +
```

Comment:
from where did you take this? seems it is not beta(FLAGS_train_p_mixup, FLAGS_train_p_mixup), no?